src/Craned/TaskManager.cpp

```
@@ -75,16 +83,44 @@ EnvMap TaskInstance::GetTaskEnvMap() const {
+  // TODO CRANE_NTASKS_PER_CORE IS NOT SET
+  uint64_t gpus_per_node = 0;
+  uint32_t alloc_node_num = this->task.node_num();
+  if (alloc_node_num != 0) {
+    gpus_per_node = this->task.total_gpus() / alloc_node_num;
+  }
+  uint64_t mem_in_node =
+      this->task.resources().allocatable_res_in_node().memory_limit_bytes() /
+      (static_cast<uint64_t>(1024 * 1024));
+
+  double cpus_on_node =
+      this->task.resources().allocatable_res_in_node().cpu_core_limit();
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)